src/pages/index.js

```diff
@@ -42,18 +64,77 @@ const modalTitle = document.querySelector('.modal__description');
 /*                                   Classes                                  */
 /* -------------------------------------------------------------------------- */
 
+//Api
+const api = new Api({
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+        authorization: "2d74b34c-f524-4ec8-a70b-ae8ff175319b",
+        "Content-Type": "application/json"
+    }
+});
+//
+
 //Section
 const section = new Section(
-    {items: initialCards, renderer: (cardData) => {
+    {/*items: initialCards,*/renderer: (cardData) => {
         const cardElement = getCardElement(cardData);
         section.addItem(cardElement)
-    }},
+     }},
     '.cards__list'
 );
+//
 
-section.renderItems();
+//UserInfo
+const userInfo = new UserInfo(profileName, profileBio, profileImage);
 //
 
+api
+    .getUserInfo()
+    .then((data) => {
+        console.log(data); ///take this out this was to make sure right thing was being called
+        const name = data.name;
+        const job = data.about;
+        const image = data.avatar
+        console.log(name);
+        console.log(job);
+        userInfo.setUserInfo(name, job);
+        userInfo.setUserPicture(image);
+    })
+    .catch((err) => {
+        console.error(err);
+      });
+
+/*api
+    .getInitialCards()
+    .then((res) => { //might need parameter of res, if so replace initialCards with res
+        const section = new Section(
+            {items: res, renderer: (cardData) => {
+                const cardElement = getCardElement(cardData);
+                section.addItem(cardElement)
+            }},
+            '.cards__list'
+        );
+        section.renderItems();
+    })
+    .catch((err) => {
+        console.error(err);
+    });*/
+
+api
+    .getInitialCards()
+    .then((cards) => {
+        /*cards.forEach(cardData => {
+            section.renderItems(cardData); //use renderItems(section)
+        });*/
+        section._items = cards;
```

Comment:
`_items` 


Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.